test/test_install_kubernetes_full.sh

```diff
@@ -46,9 +46,6 @@ auth_at_keptn $KEPTN_ENDPOINT $KEPTN_API_TOKEN
 
 verify_test_step $? "Could not authenticate at Keptn API"
 
-echo "Keptn installed in version:"
```

Comment:
Could we leave the version check here to test the functionality of getting the `Keptn cluster version` after a successful installation?

```
>keptn version

Keptn CLI version: 0.8.0
Keptn cluster version: 0.8.0
```